### ISSUE TITLE
Outdated typescript peer dependency

### ISSUE THREAD

**Issue by chet-manley** [angular-extensions/model#5]
## Minimal reproduction of the bug with instructions:
Running `ng add @angular-extensions/model` generates the following warning:
```
npm WARN @angular-extensions/model@7.0.1 requires a peer of typescript@~3.1.0 but none is installed. You must install peer dependencies yourself.
```
Running `ng update --all` generates:
```
Package "@angular-extensions/model" has an incompatible peer dependency to "typescript" (requires "~3.1.0", would install "3.2.4").
```
The typescript peer dependency used by @angular/angular is as follows:
```
"typescript": "~3.2.2"
```
So Angular installs typescript 3.2.4, which breaks your modules peer dependency. I do not believe there were any breaking changes from typescript ~3.1 to ~3.2, so any easy fix would be changing your dependency accordingly, at least to match what current versions of Angular are using. 

## Expected behavior:
Installing or updating this package should install peer dependencies in line with current versions of Angular. 

## Other information:
```
Angular CLI: 7.2.2
Node: 10.15.0
OS: linux x64
Angular: 7.2.1
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.12.2
@angular-devkit/build-angular     0.12.2
@angular-devkit/build-optimizer   0.12.2
@angular-devkit/build-webpack     0.12.2
@angular-devkit/core              7.2.2
@angular-devkit/schematics        7.2.2
@angular/cli                      7.2.2
@ngtools/webpack                  7.2.2
@schematics/angular               7.2.2
@schematics/update                0.12.2
rxjs                              6.3.3
typescript                        3.2.4
webpack                           4.28.4
```
## I would be willing to submit a PR to fix this issue:

[X] Yes (Assistance is provided if you need help submitting a pull request)  
[ ] No

**Comment by tomastrajan** [comment]
Hi @chet-manley thank you for the great catch! Would you be willing to submit a PR that will relax the `typescript` peer dependency version to match the minor version using `^` ?

**Comment by tomastrajan** [comment]
Fixed 58f2268576cf252163e3b116428638a9dc220e13

**Comment by chet-manley** [comment]
Sorry man. I got so wrapped up in this other project that I missed your `@`. Thanks for the quick reply and fix!
